Configuration/TCA/tx_blog_domain_model_author.php

```diff
@@ -253,8 +253,9 @@
             'config' => [
                 'type' => 'select',
                 'renderType' => 'selectMultipleSideBySide',
-                'multiple' => 1,
+                'multiple' => 0,
```

Comment:
Why do you change the multiple setting? a post can have multiple authors as well as one author can have multiple posts?

Reply:
seems to be a mistake

Reply:
fixed